Source/SwiftLintFramework/Rules/ColonRule.swift

```diff
@@ -15,6 +15,16 @@ public struct ColonRule: CorrectableRule, ConfigurationProviderRule {
 
     public init() {}
 
+    public var flexibleRightSpacing = false
+
+    public init(configuration: AnyObject) throws {
+        self.flexibleRightSpacing = configuration["flexible_right_spacing"] as? Int == 1
```

Comment:
Doesn't this work?

```swift
self.flexibleRightSpacing = configuration["flexible_right_spacing"] == true
```

If so, that's a more accurate representation of what we're checking, no?